src/Distributions.jl

```diff
@@ -230,6 +234,7 @@ export
     rate,               # get the rate parameter
     sampler,            # create a Sampler object for efficient samples
     scale,              # get the scale parameter
+    scale!,             # provide storage for the scale parameter (used in multivariate distribution mvlognormal)
```

Comment:
This seems like a defensibly similar usage of "scale" to me, so I would just extend Base's versions of these.